### DISCUSSION TITLE
How to scroll to only one panel on swipe

### DISCUSSION THREAD

**Discussion by eddiedane** [naver/egjs-flicking#563] [Q&A] [answered]
Hi, just discover flicking, really exciting,
I have been trying to implement scrolling only one panel on every swipe,
I would not like the panel not to free-scroll to the end when the user swipe with force,
only display one panel at a time for every swipe,

Please can I implement this feature with flicking-v4, as I have tried various combination of different available options.
Cheers!

**Comment by WoodNeck** [comment, marked as answer]
Hello @eddiedane! 
Try using `moveType: "strict"` :)

```js
import { MOVE_TYPE } from "@egjs/flicking";

const flicking = new Flicking("#el", {
  // Detailed
  moveType: [MOVE_TYPE.STRICT, { count: 1 }],
  // Simplified
  moveType: MOVE_TYPE.STRICT,
  moveType: "strict"
});
```
